tabakins2html.py

```diff
@@ -0,0 +1,125 @@
+from railroad import Diagram, Choice, Comment, NonTerminal, Sequence, Terminal, Skip, Optional, ZeroOrMore, OneOrMore
+import sys
+import pymustache
+import io
+
+ALLOWED_NAMES = {
+    "Comment": Comment,
+    "Diagram": Diagram, 
+    "Choice": Choice, 
+    "Comment": Comment, 
+    "NonTerminal": NonTerminal, 
+    "Sequence": Sequence, 
+    "Terminal": Terminal,
+    "Skip": Skip,
+    "Optional": Optional,
+    "ZeroOrMore": ZeroOrMore,
+    "OneOrMore": OneOrMore,
+    "undefined": 'undefined'
+}
+
+def eval_expression(input_string, allowed_names = {}):
+    code = compile(input_string, "<string>", "eval")
+    for name in code.co_names:
+        if name not in allowed_names:
+            raise NameError(f"Use of {name} not allowed")
+    return eval(code, {"__builtins__": {}}, allowed_names)
+
+def pars_tadatkins_file():
+    dump = []
+    current = []
+    with open( 'pop11_grammar_tabatkins.txt', 'r' ) as file:
+        for line in file:
+            if line.startswith('//'):
+                if current:
+                    dump.append( current )
+                    current = []
+            else:
+                current.append( line )
+        if current:
+            dump.append( current )
+    return map( lambda L: ''.join(L), dump )
+
+def diagram2string( diag ):
+    with io.StringIO() as buffer:
+        diag.writeSvg(buffer.write)   
+        return buffer.getvalue()
+
+def svgList():
```

Comment:
svgList -> svg_list to be consistent within file.

Reply:
Fixed